sdk/keyvault/Azure.Security.KeyVault.Administration/src/KeyVaultBackupClient.cs

```diff
@@ -185,6 +185,70 @@ public virtual RestoreOperation StartRestore(Uri blobStorageUri, string sasToken
             }
         }
 
+        /// <summary>
+        /// Initiates a selective restore of the Key Vault.
+        /// </summary>
+        /// <param name="keyName">The name of the key to be restored from the supplied backup.</param>
+        /// <param name="blobStorageUri">The Uri for the blob storage resource.</param>
```

Comment:
Nit: if referring to the type `Uri`, please use `<see cref="Uri"/>`; otherwise, use the acronym "URI".